pysc2/pysc2/agents/base_agent.js

```diff
@@ -0,0 +1,32 @@
+const path = require('path')
+const actions = require(path.resolve(__dirname, '..', 'lib', './action.js'))
+
+class BaseAgent { //eslint-disable-line
+
+	constructor(){ //eslint-disable-line
+    this.reward = 0
+    this.episodes = 0
+    this.steps = 0
+    this.obs_spec = 0
```

Comment:
lines 10 & 11 use null instead of 0

In general Python None is JavaScript Null